vertx-web/src/main/java/io/vertx/ext/web/templ/TemplateEngine.java

```diff
@@ -41,6 +43,21 @@
   void render(RoutingContext context, String templateFileName, Handler<AsyncResult<Buffer>> handler);
 
   /**
+   * Shortcut for rendering to the provided context with text/html content type
+   * @param context the routing context to render to
+   * @param templateFileName the template file name to use
+   */
+  default void render(RoutingContext context, String templateFileName) {
+    render(context, templateFileName, (res) -> {
+      if (res.succeeded()) {
+        context.response().putHeader(HttpHeaders.CONTENT_TYPE, TemplateHandler.DEFAULT_CONTENT_TYPE).end(res.result());
```

Comment:
I actually think the API belongs in TemplateHandler and not TemplateEngine. I will be moving it.